Tensile/Source/client/main.cpp

```diff
@@ -124,7 +125,7 @@ namespace Tensile
                 ("init-d",                   po::value<InitMode>()->default_value(InitMode::Zero), "Initialization for D")
                 ("init-alpha",               po::value<InitMode>()->default_value(InitMode::Two), "Initialization for alpha")
                 ("init-beta",                po::value<InitMode>()->default_value(InitMode::Two), "Initialization for beta")
-                ("pristine-on-gpu",          po::value<bool>()->default_value(false), "Keep a pristine copy of inputs on GPU for performance")
+                ("pristine-on-gpu",          po::value<bool>()->default_value(true), "Keep a pristine copy of inputs on GPU for performance")
```

Comment:
As @sdquiring mention, use pin memory can faster verify part.
All our testing is pass, some of it contain quit large size.

Reply:
Possible, although it does speed things up considerably to have it enabled.  We should probably hook this up to a `globalParameter` in case this does break something.

Reply:
does it mean we have to set default back to false.  And connect it with globalParameter later?

Reply:
I think we could just add a globalParameter now.

Reply:
done with connect to globalParameter